src/App.tsx

```diff
@@ -1,23 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
create CONSTANTS for initial values

Reply:
@ArtemTeslenko This is not the case of [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)), because name of the state, and setState methods clearly indicates, what is being stored in them. Also, default perPage integer does not reoccur anywhere else in the code, hence, using it in the properly named state is completely justified.

Reply:
👀